### PR TITLE
Userguide: Wrong param order in chmod command

### PR DESCRIPTION
**Pull request by andreascreten**

### CODE REVIEW

File: guide/kohana/install.md [hunk 1]
```diff
@@ -22,8 +22,8 @@
 6. Make sure the `application/cache` and `application/logs` directories are writable by the web server.
 
 	~~~
-		sudo chmod 777 -R application/cache
-		sudo chmod 777 -R application/logs
+		sudo chmod -R 777 application/cache
```
Comment (shadowhand) [inline guide/kohana/install.md]: Since you edited this, `777` should probably be `0777` to be completely accurate.

Comment (sergeyklay) [inline guide/kohana/install.md]:
``` sh
sudo chmod -R a+rwx application/cache
```

See: [man chmod](http://en.wikipedia.org/wiki/Chmod)

Comment (shadowhand) [inline guide/kohana/install.md]: That's probably even better.

Comment (andreascreten) [inline guide/kohana/install.md]: Made a new commit to fix those.

**Comment by zeelot** [comment]
Thanks, merged it in manually but please file a ticket on redmine next time and make the pull request to our develop branches instead of master, we can't merge anything into master :)

Cheers

**Pull request closed without merging**